dist/cronstrue-i18n.js

```diff
@@ -1127,6 +1127,8 @@ var fa_1 = __webpack_require__(35);
 Object.defineProperty(exports, "fa", { enumerable: true, get: function () { return fa_1.fa; } });
 var ca_1 = __webpack_require__(36);
 Object.defineProperty(exports, "ca", { enumerable: true, get: function () { return ca_1.ca; } });
+var be_1 = __webpack_require__(37);
```

Comment:
No need to update the `dist/` folder files as these will get generated.  But, I don't think it will hurt anything; they will just get updated when I build the release.